scripts/ctool

```diff
@@ -969,11 +973,17 @@ delete_container() {
     lxc delete --force "$1"
 }
 
+while [ "${1:-}" = "-v" -o "${1:-}" = "--verbose" ]; do
+    VERBOSITY=$((VERBOSITY+1));
```

Comment:
I think this could just be:
```
((VERBOSITY++))
```

Reply:
well, its the  other way everywhere else.